### PR TITLE
Feat/html structure

### PR DESCRIPTION
**Pull request by borislavkb**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -8,18 +8,16 @@
...
   </head>
   <body>
-    <header></header>
+    <header><h1 class="heading">BMI Calculator</h1></header>
     <main>
       <section class="bmi-container">
-        <div class="bmi-box">
-          <h1 class="heading">BMI Calculator</h1>
-        </div>
+        <div class="bmi-box"></div>
```
Comment (mpagels) [inline index.html]: Didn't look into the whole code, just the pr. Why are here a div without content? Do you plan to fill it with content with dynamically with JS?

**Review by mpagels — APPROVED** [review]
Lovely small pull request. Looking good!

File: index.html (hunk 1 above)
Comment (borislavkb) [reply index.html]: I guess I missed this when moved the ```h1``` . Thx !

**Review by borislavkb — COMMENTED** [review]
Remove a div element. 
Inspect changes.

**Pull request merged**